app/services/timeline_events/create_service.rb

```diff
@@ -11,7 +11,12 @@ def execute
         TimelineEvent.create!(@params).tap do |te|
           @founder.timeline_event_owners.create!(timeline_event: te, latest: true)
           create_team_entries(te) if @params[:target].team_target?
-          WebhookDeliveries::CreateService.new(@founder.course, WebhookDelivery.events[:submission_created]).execute(te)
+          Developers::NotificationService.new.execute(
+            @founder.course,
+            WebhookDelivery.events[:submission_created],
+            @founder,
```

Comment:
How about always sending a `User`? This would make it safer, since the _type_ would be fixed.

Reply:
Yes, and less complicated. Will change it.